repo.lb

```diff
@@ -228,10 +228,15 @@ def init(repo):
             path = normpath(path)
             escaped = "\\" * (2 ** escape_level)
             return path.replace("\\", escaped)
+        def posixify(path):
+            return normpath(path).replace("\\", "/")
```

Comment:
I feel like this is a dumb solution, and that lbuild should have some option to reformat paths somehow, but it's all such a horrible mess, I don't want to deal with it.